build/pulsar_ci_tool.sh

```diff
@@ -579,6 +579,59 @@ ci_create_inttest_coverage_report() {
   echo "::endgroup::"
 }
 
+ci_report_netty_leaks() {
+  if [ -z "$NETTY_LEAK_DUMP_DIR" ]; then
+    echo "NETTY_LEAK_DUMP_DIR isn't set"
+    return 0
+  fi
+  local temp_file=$(mktemp -t netty_leak.XXXX)
+
+  # concat all netty_leak_*.txt files in the dump directory to a temp file
+  if [ -d "$NETTY_LEAK_DUMP_DIR" ]; then
+    find "$NETTY_LEAK_DUMP_DIR" -maxdepth 1 -type f -name "netty_leak_*.txt" -exec cat {} \; >> $temp_file
+  fi
+
+  # check if there are any netty_leak_*.txt files in the container logs
+  local container_logs_dir="tests/integration/target/container-logs"
+  if [ -d "$container_logs_dir" ]; then
+    local container_netty_leak_dump_dir="$NETTY_LEAK_DUMP_DIR/container-logs"
+    mkdir -p "$container_netty_leak_dump_dir"
```

Comment:
do we need to clean this dir from the previous runs?

Reply:
In GitHub Actions CI, there will be only a single build and everything gets cleaned up after that. These scripts are for GitHub Actions.